Stubs/Xamarin.Forms.Platform.Tizen/Xamarin.Forms.Platform.Tizen (Forwarders).csproj

```diff
@@ -2,16 +2,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>tizen40</TargetFramework>
-    <Configurations>Debug;Release</Configurations>
-    <Platforms>AnyCPU</Platforms>
-    <ProjectGuid>{39B3457F-01D8-43D0-8E84-D8C4F73CF48E}</ProjectGuid>
-    <ProjectTypeGuids>{FEACFBD2-3405-455C-9665-78FE426C6842};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <RootNamespace>Xamarin.Forms.Platform</RootNamespace>
     <AssemblyName>Xamarin.Forms.Platform</AssemblyName>
     <DefineConstants>TIZEN4_0</DefineConstants>
     <OutputType>Library</OutputType>
     <PackageId>Xamarin.Forms.Platform.TizenForwarders</PackageId>
-    <GenerateDependencyFile>false</GenerateDependencyFile>
```

Comment:
If I recall correctly, this is required to avoid some error mentioned in #2661. I'll double check it out.